Sources/TuistLoader/Models+ManifestMappers/Target+ManifestMapper.swift

```diff
@@ -30,6 +30,10 @@ extension TuistCore.Target {
             return (glob: globPath, excluding: excluding, compilerFlags: glob.compilerFlags)
         } ?? [])
 
+        if sources.isEmpty {
```

Comment:
Mappers' responsibility should be mapping. If you want to print a warning or error because there are no sources, I'd recommend moving the logic to the [inter](https://github.com/tuist/tuist/blob/master/Sources/TuistGenerator/Linter/TargetLinter.swift)